code/game/objects/items/devices/multitool.dm

```diff
@@ -53,18 +53,60 @@
 				for(var/mob/camera/aiEye/A in chunk.seenby)
 					var/turf/eye_turf = get_turf(A)
 					if(get_dist(our_turf, eye_turf) < 8)
+						found_eye = 2
+						break
+					if(get_dist(our_turf, eye_turf) < 20) //AI is near you, be careful
 						found_eye = 1
 						break
 
-	if(found_eye)
+	if(found_eye == 1)
+		icon_state = "[initial(icon_state)]_yellow"
+	else if(found_eye == 2)
 		icon_state = "[initial(icon_state)]_red"
 	else
 		icon_state = initial(icon_state)
 
 	track_delay = world.time + 10 // 1 second
 	return
 
+/obj/item/device/multitool/admin_detect
+	desc = "Used for pulsing wires to test which to cut. Not recommended by doctors. Has a strange tag that says 'Grief in Safety'" //What else should I say for a meme item?
+	var/track_delay = 0
+
+/obj/item/device/multitool/admin_detect/New()
+	..()
+	SSobj.processing += src
+
+/obj/item/device/multitool/admin_detect/Destroy()
+	SSobj.processing -= src
+	return ..()
+
+/obj/item/device/multitool/admin_detect/process()
+	if(track_delay > world.time)
+		return
 
+	var/found_admin = 0
+	var/turf/our_turf = get_turf(src)
+
+	for(var/mob/J in range(20,src))
+		if(admin_datums[J.ckey])
+			var/turf/admin_turf = get_turf(J)
+			if(get_dist(our_turf, admin_turf) < 8)
+				found_admin = 2 //Oh shit.
+				break
+			if(get_dist(our_turf, admin_turf) < 20)
+				found_admin = 1 //Uh oh, gotta watch out.
+				break
+
+	if(found_admin == 1)
+		icon_state = "[initial(icon_state)]_yellow"
+	else if(found_admin == 2)
+		icon_state = "[initial(icon_state)]_red"
+	else
+		icon_state = initial(icon_state)
+
+	track_delay = world.time + 5 //Gotta be extra careful here

```

Comment:
On further inspection this proc is pretty much all copy paste
This comment is saying that you've gotta be extra careful compared to the AI multitool, so it processes twice as often